wiki/Project_Loved/en.md

```diff
@@ -28,6 +28,7 @@ There are some very minimal criteria that need to be met for beatmaps to be nomi
 - Its creator is neither banned nor restricted
 - At least one difficulty can be passed
 - It has at least 30 favourites (only applies to osu!standard maps)
+- Every difficulty of a beatmapset must have a minimum [drain time](/wiki/en/Gameplay/Drain_time) of 30 seconds.
```

Comment:
```suggestion
- Every difficulty of a beatmapset must have a minimum [drain time](/wiki/Gameplay/Drain_time) of 30 seconds.
```
The link should not have the `en/` part. Also, are we also moving away from using beatmapset?

Reply:
I copied and pasted this exact point from the [Ranking Criteria](https://osu.ppy.sh/wiki/en/Ranking_Criteria) page so I'm not entirely sure.

Reply:
> are we also moving away from using beatmapset?

we totally are, but I need to organize the existing `Beatmap`/`Beatmapset` articles first, then play find-and-replace